### ISSUE TITLE
Icon request - Withings

### ISSUE THREAD

**Issue by jonathanathe** [simple-icons/simple-icons#5604]
**Brand Name:** Withings
**Website:**https://www.withings.com/
**Alexa rank:** 36,234
  

**Official resources for icon and color:**
  
  https://www.withings.com/it/en/press#logo

**Comment by PeterShaggyNoble** [comment]
Thanks for the request, @jonathanathe. Note that the logo available for download on the page you linked is a PNG. But there _is_ [an SVG](https://www-assets.withings.com/site/media/logo/withings-logotype-black.svg) available in the website header.

Looks like we'll need to go with the full wordmark here. Would you like to take it on yourself?

**Comment by jonathanathe** [comment]
Hi @PeterShaggyNoble,  sorry for my English, would that mean I would have to do the whole process as I did in this ticket? #5440

**Comment by jonathanathe** [comment]
I would like this icon to be there, but I'm still too much of a novice to remember or know how to do it.

My question is, how do I know what to delete on this "Select all the paths that aren't part of the logo and delete them.

And where do I find this "Open the XML editor (Edit - >XML Editor or Ctrl + Shift + X) and expand all the nodes until you find the lone svg:path one."


![image](https://user-images.githubusercontent.com/34164094/117247032-e00b3100-ae3d-11eb-9edc-cb1f1b28d3a9.png)

**Comment by jonathanathe** [comment]
Find this logo too:


https://upload.wikimedia.org/wikipedia/commons/b/b1/Withings_logo.svg

**Comment by PeterShaggyNoble** [comment]
The icon in that logo would definitely suit us better, @jonathanathe but, looking at [their Wikipedia page](https://en.wikipedia.org/wiki/Withings) its not in use there so it looks like that is an outdated logo.

This one is actually a lot more straightforward than the last icon we worked on together so the step-by-step tutorial I provided you with isn't applicable here. All that needs to be done with this one is:
1. Resize the canvas.
2. Ungroup the paths
3. Merge them
4. Resize that merged path
5. Centre it on the canvas
6. Optimise

The _only_ thing is that the "G" & the "S" are a mess of points, probably a result of text having been converted to a path somewhere in the process of the creation of the original file. We'd need to clean those up as best we could but it's going to take someone with better SVG skills than I to help you out with that so I suggest going straight to PR once you've completed the steps above,.
